src/main/java/duke/parser/Parser.java

```diff
@@ -0,0 +1,93 @@
+package duke.parser;
+
+import duke.command.FindCommand;
+import duke.command.ListCommand;
+import duke.command.AddCommand;
+import duke.command.DoneCommand;
+import duke.command.DeleteCommand;
+import duke.command.ByeCommand;
+import duke.command.Command;
+
+import duke.exception.DukeException;
+
+/**
+ * Represents a parser that parses a line, returning a Command instance.
+ */
+public class Parser {
+    private static final String unknownCommand = "☹ OOPS!!! I'm sorry, but I don't know what that means :-(";
```

Comment:
Kindly consider having your constant names be all uppercase, using underscore to separate words.